share/spice/currency/currencyNames.txt

```diff
@@ -156,7 +156,7 @@ vuv,ni-vanuatu vatu,vanuatu vatu,
 wst,samoan tala,samoa tala,
 xaf,central african cfa franc beac,re africaine cfa franc beac,
 xag,silver ounce,silver,
-xau,gold ounce,gold,
+xau,gold ounce,gold,1 oz gold in usd,gold price,gold prices today,gold price per ounce,1oz gold,1oz gold in usd,1 ounce gold in usd, 1 ounces gold in usd
 xbt,bitcoin,btc,bitcoins
```

Comment:
@MrChrisW I Agree. I was about to make the same comment :)

This is kind of a "brute force" approach -- it works, but it isn't very flexible

`1 oz` is a prefix we can expect for other metals as well -- the correct solution should be allow us to handle other precious metals as well. E.g. `1 oz platinum` should work, but right now only `platinum` works:
https://duckduckgo.com/?q=platinum&ia=currency

Also, if the API allows, it would be great to handle queries like `X oz gold/platinum/etc` (e.g. "2 oz gold") or "X oz gold/platinum/etc in USD/CAD/AUD/etc`